x/vesting/utils_test.go

```diff
@@ -1,162 +0,0 @@
-package vesting_test
```

Comment:
Why all this file has been removed? It was present in the PR adding EthAccount

Reply:
this is just the old test setup, is not beeing used in the testing suite anymore.